Grabber.pro

```diff
@@ -33,8 +33,7 @@ unix:!macx{
 	desktop.files += release/Grabber.desktop
 
 	icon.path = $$PREFIX/share/pixmaps
-	icon.extra = cp resources/icon.png resources/Grabber.png
-	icon.files += resources/Grabber.png
+	icon.extra = install -D -m644 resources/icon.png $(INSTALL_ROOT)/$$PREFIX/share/pixmaps/Grabber.png

```

Comment:
I'm just wondering what happens if the user sets the prefix to let's say "/tmp/Grabber", expecting it to install it there. I don't know what INSTALL_ROOT is for, from what I read online it seems to have the same objective as PREFIX, but special for Qt? In that case, wouldn't mixing both be weird?

If Qt sets install root to "/usr/local" and the user set the prefix to "/tmp/Grabber", the program will be installed in "/usr/local/tmp/Grabber".

Or maybe I missed something, I haven't been on Linux in a while.